src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -1200,8 +1200,7 @@ public void mutateRepaired(Collection<SSTableReader> sstables, long repairedAt,
         {
             for (SSTableReader sstable: sstables)
             {
-                sstable.descriptor.getMetadataSerializer().mutateRepairMetadata(sstable.descriptor, repairedAt, pendingRepair, isTransient);
-                sstable.reloadSSTableMetadata();
+                sstable.mutateAndReloadStats(stats -> stats.mutateRepairedMetadata(repairedAt, pendingRepair, isTransient));
```

Comment:
@jasonstack Do we lose trace recording if we call `mutate()` directly rather than `IMetadataSerializer#mutateRepairMetadata()`?

Reply:
I can put the trace back.. or revert the change if stats component is preloaded.

Reply:
@jasonstack Ah, just a reminder to put this back if possible before we commit.

Reply:
added an `description` argument to pass trace info